distributed/tests/test_client.py

```diff
@@ -1694,7 +1694,7 @@ def test_repr(loop):
             for func in funcs:
                 text = func(c)
                 assert c.scheduler.address in text
-                assert '2' in text
```

Comment:
This would succeed simply because the address of the scheduler is always `tcp://127.0.0.1:something` ;-)